chsdi/models/vector/bafu.py

```diff
@@ -1098,19 +1098,23 @@ class Gewaesserschutzbereiche (Base, Vector):
 
 
 class Vorfluter (Base, Vector):
-    __tablename__ = 'vorfluter'
+    __tablename__ = 'effluent'
     __table_args__ = ({'schema': 'wasser', 'autoload': False})
     __bodId__ = 'ch.bafu.wasser-vorfluter'
     __template__ = 'templates/htmlpopup/vorfluter.mako'
-    __label__ = 'teilezgnr'
+    __label__ = 'id'
     id = Column('bgdi_id', Integer, primary_key=True)
-    teilezgnr = Column('teilezgnr', Integer)
+    id_2 = Column('id', Integer)
```

Comment:
why do you need to expose both ids bgdi_id and id (from the data , which is primary key). i would only expose the id which comes with the data

Reply:
ID is an attribute from the table delivered by bafu and must apparear in the tooltip as specified in the data integration form.

However as they didn't specifiy a stabile id in the data integration form to be used, I created one with the table (the bgdi_id)

Reply:
Ok I will use these id. thanks for the feedback